src/olympia/devhub/tasks.py

```diff
@@ -143,17 +145,20 @@ def wrapper(task, id_, hash_, *args, **kw):
 
 
 @validation_task
-def validate_file_path(path, hash_, listed, **kw):
+def validate_file_path(path, hash_, listed=True, is_webextension=False, **kw):
```

Comment:
Are you sure about this default value of `True` for `is_listed`? Are all the calls to `validate_file_path` expecting that?